imap_processing/cdf/utils.py

```
@@ -135,12 +137,16 @@ def write_cdf(
         ]
 
     # Convert the xarray object to a CDF
-    xarray_to_cdf(
-        dataset,
-        str(file_path),
-        terminate_on_warning=True,
-        **extra_cdf_kwargs,
-    )  # Terminate if not ISTP compliant
+    if "l1" in data_level:
+        if "istp" not in extra_cdf_kwargs:
+            extra_cdf_kwargs["istp"] = False  # type: ignore
```

Added ignore this error
```
map_processing/cdf/utils.py:140: error: Incompatible types in assignment (expression has type "bool", target has type "dict[Any, Any]")  [assignment]
```

---

Weird. why is bool not a part of Any 🤷 

Did you have to add the `dict[Any, Any]` or could you just leave it alone above on entry into the function?

---

Yeah, mypy <sigh>